google_shopping.js

```diff
@@ -1,5 +1,54 @@
 var data = require("./products.json")
 
 // Write your solutions below
+var q = 0
+for (i = 0; i < data["items"].length; i++){
+
+	if (data["items"][i].product.inventories[0].availability === "inStock"){
+		q++;
+	}
+	
+}
+console.log("There are " + q + " items In Stock.");
```

Comment:
I like this method! Alternatively, you can measure how many items are on backorder using 
```data["items"][i].product.inventories[0].availability === 'backorder'```